gems/pending/appliance_console.rb

```diff
@@ -94,8 +96,24 @@
 require 'appliance_console/prompts'
 include ApplianceConsole::Prompts
 
+# Updates the ip address associated with a hostname in the /etc/hosts file
+#
+# @param host [String] The hostname to look for
+# @param ip [String] The address to write
+def update_hostname_ip(host, ip)
```

Comment:
Right, would it be better to move the network operations into methods in a different module maybe a file at `gems/pending/appliance_console/network_operations.rb`?  Even some of the stuff that is only used in one place is a bit lengthy for this file.